torch/_inductor/kernel/mm.py

```diff
@@ -399,6 +400,59 @@ def _is_sm7x_or_older_gpu(index: Optional[int]) -> bool:
     return props.major <= 7
 
 
+def mixed_mm_use_heuristic(choices, mat1, mat2, mat2_dtype, layout, m, n, k, config):
```

Comment:
This refactor seems a bit unnecessary to me. I would instead just return the config from `try_heuristic` and append the choice in the main body.

Reply:
Let me know if my changes resolve your comment